test/vm_tests/transformer/impl_generation_test.dart

```diff
@@ -652,6 +731,33 @@ main() {
           });
         });
       });
+
+      group('static meta field', () {
```

Comment:
It would be sweet if we also had an integration test for these.

It looks like we don't have any for `$PropKeys`/`$Props`, but there's this test that tests them indirectly: https://github.com/Workiva/over_react/blob/master/test/over_react/component_declaration/transformer_integration_tests/do_not_generate_accessor_integration_test.dart#L44

Thoughts on adding a test there for good measure?
```dart
        expect(DoNotGenerateAccessorTestProps.meta.propKeys, [
          contains('generated1Prop'),
          contains('generated2Prop'),
          contains('explicitlyGeneratedProp'),
        ], reason: 'should only include generated props');

        expect(DoNotGenerateAccessorTestProps.meta.map((prop) => prop.key).toList(), [
          contains('generated1Prop'),
          contains('generated2Prop'),
          contains('explicitlyGeneratedProp'),
        ], reason: 'should only include generated props');
```